requirements.txt

```diff
@@ -1,10 +1,10 @@
-astroid==1.6.3
-pylint==1.8.4
+pylint==1.9.3 ;python_version < '3'
+pylint==2.1.1 ;python_version >= '3'
 pylint-plugin-utils==0.2.4
 docutils==0.14
-lxml>=2.3.2
+lxml==4.2.3
```

Comment:
Is now this library faster to be compiled?

Reply:
This one has dual compatibility py2 and py3.7
Wheels are used for old and new version I think so